### ISSUE TITLE
#+REVEAL: split changes slide ordering

### ISSUE THREAD

**Issue by helasraizam** [yjwen/org-reveal#345]
I have a set of slides such that the chapters are each a column (left-right changes chapters, up-down changes the slide within that chapter)--chapters are headings (one asterisk) and slides are subheadings (two or more asterisks).  Everything works except when I use #+REVEAL: split on a slide in a chapter, pressing up or down on that slide actually moves to the previous chapter, disrupting the flow of the slides.

My org-mode version is 9.1.14

**Comment by helasraizam** [comment]
Good news!  I've traced the issue to 3b02186, where `section-number` was introduced.  I don't think this function (defined on line 440) does what it's supposed to do; as shown above, the `<section>` tags don't have unique ids in my case, all ids default to `slide-sec-` without a number at the end.  So the problem isn't that the id isn't prepended to the sections after the split, but that the ids aren't unique to begin with.  Replacing the `section-number` call on line 442 of ox-reveal.el with `(org-export-get-reference headline info)` fixes the issue, with the unique section ids now reading `<section id="slide-sec-org3a9a1a1">`, whereas before they all read `<section id="slide-sec-">`, and sections still appearing without ids after the splits.  Turns out #277 was just forked before 3b02186, which led me to the error.
